### PR TITLE
Implement aidpackage img link edit functionality

### LINKED ISSUE #479: Upload donor logo with aid package
- [ ] Upload donor logo with aid package which is publicly visible in the donor portal/ public view. 

Rationale: to create more of a feeling that the platform is a coordinating platform for all organizations and not only red cross.

### PR DESCRIPTION
**Pull request by dinuka-rp**
## Purpose

Admin portal part of, fix #479

I'm not proud of the way this was implemented. The code needs to be refactored to make use of reusable components.

_**Both Banner & Thumbnail image update work #479**_

### CODE REVIEW

**Comment by dinuka-rp** [comment]
Banner image update works. Waiting on backend updates to support thumbnail image

image

image

File: admin-portal/src/pages/packageDetails/packageDetails.tsx [hunk 1]
```diff
@@ -140,16 +146,36 @@ export default function PackageDetails() {
...
       description: editedAidPackage.description,
     });
-    await fetchAidPackage();
+    await fetchAidPackage(); // FIXME: why isn't this just updated from the info that exists in the frontend, based on the response received from the backend?
     setIsEditDescriptionModalVisible(false);
   };
 
+  const handleBannerUrlUpdate = async (editedAidPackage: AidPackage) => {
+    await AidPackageService.updateAidPackage(editedAidPackage);
+    // await fetchAidPackage(); // FIXME: why isn't this just updated from the info that exists in the frontend, based on the response received from the backend?
```
Comment (anjula-sack) [inline admin-portal/src/pages/packageDetails/packageDetails.tsx:162]: Is this fetch required?

Comment (anjula-sack) [reply admin-portal/src/pages/packageDetails/packageDetails.tsx:162]: If not let's remove the whole thing

Comment (anjula-sack) [inline admin-portal/src/pages/packageDetails/packageDetails.tsx:162]: Let's remove this if it's not required

File: admin-portal/src/pages/packageDetails/packageDetails.tsx [hunk 2]
```diff
@@ -140,16 +146,36 @@ export default function PackageDetails() {
...
+  };
+
+  const handleThumbnailUrlUpdate = async (editedAidPackage: AidPackage) => {
+    console.log("handleThumbnailUrlUpdate");
+    await AidPackageService.updateAidPackage({
+      ...aidPackage!,
+      status: aidPackageStatus!,
+      thumbnail: editedAidPackage.thumbnail,
+    });
+    // await fetchAidPackage(); // FIXME: why isn't this just updated from the info that exists in the frontend, based on the response received from the backend?
```
Comment (anjula-sack) [inline admin-portal/src/pages/packageDetails/packageDetails.tsx:174]: This as well

File: admin-portal/src/pages/packageDetails/packageDetails.tsx (hunk 1 above)
Comment (dinuka-rp) [reply admin-portal/src/pages/packageDetails/packageDetails.tsx]: ok 👍🏾

**Comment by dinuka-rp** [comment]
Thumbnail image upload works.

image

image

**Comment by dinuka-rp** [comment]
@nshamit Is it necessary to display the thumbnail image inside the package details page (where the banner is shown) of the donor portal? If so, where should I place it?

**Review by anjula-sack — APPROVED** [review]
Thank you! @dinuka-rp

**Review by anjula-sack — APPROVED** [review]

**Pull request merged**